web_widget_one2many_tags/static/src/js/web_widget_one2many_tags.js

```diff
@@ -0,0 +1,213 @@
+//-*- coding: utf-8 -*-
+//?? 2016 Therp BV <http://therp.nl>
+//License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+odoo.define('web_widget_one2many_tags.form_widgets', function (require) {
+'use strict';
+var widget_one2many_tag = require('web.Model');
+var One2manyTag = widget_one2many_tag.FieldOne2ManyTags.extend({
+        template: "FieldOne2ManyTags",
+        tag_template: "FieldOne2ManyTag",
+        disable_utility_classes: false,
+        initialize_texttext: function()
+        {
+            var self = this;
+
+            return {
+                plugins: 'tags arrow filter',
+                ext: {
+                    itemManager: {
+                        itemToString: function(item) {
+                            return item.name;
+                        },
+                    },
+                    arrow: {
+                        onArrowClick: function(e)
+                        {
+                            var list_view = new widget_one2many_tag.web.form.One2ManyListView(
```

Comment:
this a absolutely not the correct way to refer to objects that were accessed via `instance` in earlier versions. You need to `require` every object used above where you also require `Model`. And the canonical name then is the last dotted chunk.
You do this all over the place in this file, and need to change it everywhere